apps/platform/utils/mail/passwordRecoveryEmailTemplate.ts

```diff
@@ -0,0 +1,58 @@
+export type PasswordRecoveryEmailProps = {
+  to: string;
+  username: string;
+  recoveryCode: string;
+  expiryDate: string;
+};
+
+export function passwordRecoveryEmailTemplate({
+  username,
+  recoveryCode,
+  expiryDate
+}: PasswordRecoveryEmailProps): string {
+  return `
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Password Recovery for Your Uninbox Account</title>
+    </head>
+    <body style="font-family: Arial, sans-serif; line-height: 1.6; color: #333;">
+      <div style="max-width: 600px; margin: 0 auto; padding: 20px;">
+        <h1 style="color: #4a4a4a;">Password Recovery</h1>
+        <p>Hello ${username},</p>
+        <p>We received a request to reset the password for your Uninbox account. If you didn't make this request, please ignore this email.</p>
+        <p>To reset your password, please use the following recovery code:</p>
+        <p style="font-size: 24px; font-weight: bold; text-align: center; padding: 10px; background-color: #f0f0f0; border-radius: 5px;">${recoveryCode}</p>
+        <p>This code will expire on ${expiryDate}.</p>
+        <p>If you didn't request a password reset, please contact our support team immediately.</p>
+        <p>Best regards,<br>The Uninbox Team</p>
+      </div>
+    </body>
```

Comment:
This template doesn't look like a valid email to me, we actually just use pre compiled version of react email for our use,
https://demo.react.email/preview/magic-links/aws-verify-email?view=source&lang=markup
edit these templates and use the compiled html and plain text version

Reply:
didn't know we were usign that, makes sense.